### ISSUE TITLE
Uploading no selected picture on product edit page generates exception

### ISSUE THREAD

**Issue by learn2develop** [vanilophp/demo#14]
When editing product in dashboard and I do not select any picture and press upload exception
Spatie\MediaLibrary\Exceptions\FileCannotBeAdded\RequestDoesNotHaveFile appears. With message "The current request does not have a file in a key named `images`"

I am still using the demo from 2 weeks time, if it was fixed please close the issue.

**Comment by learn2develop** [comment]
Do not know if this helps https://github.com/spatie/laravel-medialibrary/issues/646

**Comment by fulopattila122** [comment]
Vanilo v0.4 was released yesterday. Please use a fresh new install and check if that works.
We've been testing it excessively in the last few weeks and haven't had this error, so it should work.

**Comment by learn2develop** [comment]
I have installed the version 0.4 and there are two problems.
First when creating new product and attempting to upload a picture there is no button to submit the picture and the picture is not uploaded with the creation of the product.

Create form - No Submit
![image](https://user-images.githubusercontent.com/6439903/48475321-72de2e00-e7f4-11e8-94b6-b9c3c3bd398f.png)
Edit form - Submit
![image](https://user-images.githubusercontent.com/6439903/48475458-bf296e00-e7f4-11e8-984f-675ed706fdf4.png)

Second problem is the one I have highlighted yesterday. When I am on product edit page and just click the green submit button to upload a picture it will throw exception.

**Comment by fulopattila122** [comment]
Hmm ok.., thanks for the report, I'll look into it shortly.

**Comment by learn2develop** [comment]
I am just having a look around the 0.4 now and the upload of files when creating products works. It it just the edit exception.

**Comment by fulopattila122** [comment]
Do you mean that the only problem is when you submit Images on product edit form without files selected with the green arrow?

**Comment by learn2develop** [comment]
Yes

**Comment by learn2develop** [comment]
I have changed ```MediaController@store```
to
```php
public function store(Request $request)
{
        if(!$request->has('images')){
            flash()->error(__('No image supplied, image is required!'));
            return redirect()->back();
        }

        // Laravel Black Magic
        $model = app(concord()->short($request->get('forType')))->find($request->get('forId'));

        $model->addMultipleMediaFromRequest(['images'])->each(function ($fileAdder) {
            $fileAdder->toMediaCollection();
        });

        flash()->success(__('Images have been added successfully!'));
        return redirect()->back();
}
```

and the exception is not triggered

**Comment by fulopattila122** [comment]
Ok, I'll add that validation to the framework

**Comment by fulopattila122** [comment]
It's been released in framework v0.4.1.
To get the update run: `composer update vanilo/framework`

Thanks again for the report + the suggestion.
